modules/infinite-scroll/infinity.css

```diff
@@ -18,6 +16,103 @@
 	padding: 6px 16px;
 }
 
+/**
+ * CSS Spinner Styles
+ */
+@keyframes spinner-inner {
+  0% { opacity: 1 }
+  100% { opacity: 0 }
+}
+.infinite-loader .spinner-inner div {
```

Comment:
Could you switch to tabs instead of spaces in this file? This way this will ensure that the next person making a change in that file does not inadvertently make changes on all lines on save.